### DISCUSSION TITLE
Cursor issue

### DISCUSSION THREAD

**Discussion by West-Tek** [ppy/osu#21042] [Q&A] [answered]
Sometimes during gampley my cursor stops moving for a split second or it directly jumps.

**Comment by mk56-spn** [comment, marked as answer]
There's so many things this could be due to, many outside of Lazer's control, that it's impossible to give you an adequate answer with so little to go on

**Reply by West-Tek** [reply]
I have been testing this issue with other games like Apex Legends or Fortnite and none of those had issues (running at ultra Grapics at 100 fps stable.
So my assumption  isthat there is somthing wrong with the game.
I let you here my Specs:  i7 2600/1660 Super/20GB DDR3/500GB SSD

**Reply by ghost** [reply]
can we have a video at the very least

**Reply by West-Tek** [reply]
i will record it this weekend if it works for you. 
and im going to try it in other pc too

**Reply by West-Tek** [reply]
> can we have a video at the very least

i have tested it in other pc and it gose just fine but i still dont know why it happens in my good system

**Reply by peppy** [reply]
Check your mouse polling rate. If it's running at something stupid like 1000hz or above, something on your system is likely tripping up on that.

**Reply by West-Tek** [reply]
Lag is fixed at the moment ty, do you have an idea on how to have a good position ( recently i dont feel confortable anymore playing osu and my wrist starts to hurt at the end of every map).
